addons/account/models/account_tax.py

```diff
@@ -113,7 +113,7 @@ class AccountTax(models.Model):
         help="The sequence field is used to define order in which the tax lines are applied.")
     amount = fields.Float(required=True, digits=(16, 4), default=0.0)
     description = fields.Char(string='Description', translate=True)
-    invoice_label = fields.Char(string='Label on Invoices')
+    invoice_label = fields.Char(string='Label on Invoices', translate=True)
```

Comment:
Doesn't this change require a migration script ? (not sure; maybe the ORM does that efficiently on its own => to check with upgrade)

Reply:
It is managed automatically
https://github.com/odoo/odoo/blob/9e228b6feb09b1afac5e7af6502b834e4755ba15/odoo/fields.py#L1668-L1673
https://github.com/odoo/odoo/blob/9e228b6feb09b1afac5e7af6502b834e4755ba15/odoo/tools/sql.py#L152-L159